digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/dao/VerificationDao.scala

```diff
@@ -264,24 +267,36 @@ class VerificationDao {
   }
 
   def getModifiedAssetTypes(linkIds : Set[Long]) : List[LatestModificationInfo] = {
-    val modifiedAssetTypes = MassQuery.withIds(linkIds) { idTableName =>
-      sql"""
-           select assetTypeId, modifiedBy, modifiedDate
-           from (
-              select a.asset_type_id as assetTypeId, a.modified_by as modifiedBy, max(TO_DATE(TO_CHAR(a.modified_date, 'YYYY-MM-DD'), 'YYYY-MM-DD hh24:mi:ss')) as modifiedDate
+    val modifiedAssetTypes =
+      if (linkIds.size > 15000) {
```

Comment:
Pitäiskö tää olla jokin vakio MassQuery eikä tälläinen ad hoc arvo?
Ihan mielenkiinnosta mikä on perustelu tälle luvulle?

Reply:
Kokeellinen arvo on tällä hetkellä, jota voidaan sitten mukauttaa testien perusteella. Voidaan kehittää vakio sitten kokemuksien perusteella. Joissakin kunnissa on melkein 50000 linkkiä ja en löytänyt mistään varmaa tietoa mikä on maksimi määrä parametreille in lauseessa, niin kokeilin tällä.

Reply:
ok, mä lisään ajastuksen tohon massquery hommaan ja yritetään etsiä joku optimaalinen arvo jossa tässä olisi jokin hyöty